docs/application/native/guides/device/system.md

```diff
@@ -26,15 +26,15 @@ To obtain the information, query a feature or system key.
 
 | [Feature keys](#feature)           |                                |                                  |                                  |
```

Comment:
I think markdown table is not good to show this table. I will change it to HTML table tag or list item.